common/src/main/java/org/apache/rocketmq/common/protocol/header/EndTransactionRequestHeader.java

```diff
@@ -121,9 +118,14 @@ public void setTransactionId(String transactionId) {
 
     @Override
     public String toString() {
-        return "EndTransactionRequestHeader [producerGroup=" + producerGroup + ", tranStateTableOffset="
-            + tranStateTableOffset + ", commitLogOffset=" + commitLogOffset + ", commitOrRollback="
-            + commitOrRollback + ", fromTransactionCheck=" + fromTransactionCheck + ", msgId=" + msgId
-            + "]";
+        return "EndTransactionRequestHeader{" +
```

Comment:
ToStringBuilder is easier way to reflect all field in one class

Reply:
I used IDEA's automatic generation tool to override the toString method.